templates_apprunner/templates_cspm_eventbridge/FullInstall.yaml

```diff
@@ -0,0 +1,87 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: IAM Role for Agentless
+Parameters:
+  SysdigRoleName:
+    Type: String
+    Default: "SysdigAgentlessRole"
```

Comment:
Can we check that this is the right default name with product? Does it match the default provisioned by the TF modules?

Reply:
checked the terraform and set the default value to `sfc-cspm-role`

Reply:
renamed it to `sysdig-secure` here and in terraform